app/tests/port_tests.c

```diff
@@ -291,6 +291,244 @@ static int ping_pong_thread(void *arg)
     return __LINE__;
 }
 
+const char *kStatusPortNames[] = {
```

Comment:
make sure all of these are static so the compiler/linker will strip them if not building with the console.

Reply:
Reposting with that change now -- thanks!

Reply:
(Some of the functions in there can be made static too and I can make/test that for this pull or just do a second. Whichever works!)